extensions/ql-vscode/test/vscode-tests/cli-integration/skeleton-query-wizard.test.ts

```diff
@@ -327,11 +327,13 @@ describe("SkeletonQueryWizard", () => {
         jest.spyOn(workspace, "workspaceFolders", "get").mockReturnValue([
           {
             name: "codeql-custom-queries-cpp",
-            uri: { path: "vscode-codeql-starter/codeql-custom-queries-cpp" },
+            uri: { fsPath: "vscode-codeql-starter/codeql-custom-queries-cpp" },
           },
           {
             name: "codeql-custom-queries-csharp",
-            uri: { path: "vscode-codeql-starter/codeql-custom-queries-csharp" },
+            uri: {
+              fsPath: "vscode-codeql-starter/codeql-custom-queries-csharp",
```

Comment:
Should these be using `join` or `path.sep` to ensure that on Windows this gets tested with a backslash instead of a slash?